configure

```diff
@@ -40,6 +42,18 @@ detect_osversion() {
     echo "$osversion"
 }
 
+# Detect OS version
+os_version=$(detect_osversion)
+
+# Check if the detected OS version is jammy or mantic
+if [[ "$os_version" == "ubuntu/jammy" || "$os_version" == "ubuntu/mantic" ]]; then
+    # Define the content to insert
+    insert_content="#include <cstdint>\n"
+
+    # Insert the content into StatType.h after the #ifndef and #define lines
+    sed -i '/^#ifndef __STATTYPE_H$/,/^#define __STATTYPE_H$/s/^/#include <cstdint>\n/' ../third_party/ramulator/src/StatType.h
```

Comment:
Again I'm not sure if overwriting in our repo is the best approach but if we do so then probably git patches are a better idea than `sed`.

Reply:
I'm actually interested in this fix to support more OS.
can you remove the runtime stuff and only submit this update standalone with configure and README.md?